mesonbuild/compilers/compilers.py

```diff
@@ -143,6 +145,13 @@ def is_library(fname):
                            'custom': [],
                            }
 
+cuda_buildtype_args = {'plain': [],
+                       'debug': [],
+                       'debugoptimized': [],
+                       'release': [],
+                       'minsize': [],
+                      }
```

Comment:
__[Flake8]__ [E124] closing bracket does not match visual indentation

<sub>posted by Sider</sub>